### ISSUE TITLE
[BUG] JSQLParser Version 4.5 : ORACLE : Encountered unexpected token 'COMMENT'

### ISSUE THREAD

**Issue by mhssgzs** [JSQLParser/JSqlParser#1727]
### SQL Example:
- Simplified Query Example, focusing on the failing feature
    ```
   alter table user modify age default 0;
    ```

### Software Information:
- JSqlParser version 4.5
- Database (Oracle)

### CodeExample:
- Simplified Query Example, focusing on the failing feature
    ```
        String sql = "alter table user modify age default 0";
        Validation validation = new Validation(Arrays.asList(DatabaseType.ORACLE), sql);
        List<ValidationError> validate = validation.validate();
    ```

### Error Message:
- Simplified Query Example, focusing on the failing feature

ValidationError [
statement=alter table user modify age default 0
capability=parsing
errors=[ParseException: Cannot parse statement: net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "default" "DEFAULT"
    at line 1, column 29.

Was expecting:

    "COMMENT"
]
]

**Comment by wu52894232** [comment]
anyone can help me answer?

**Comment by manticore-projects** [comment]
Greetings.

This specific syntax is not supported yet (for no good reason).
Please see the [`ALTER TABLE...` statement](https://jsqlparser.github.io/JSqlParser/syntax.html#altertable) which seem to have grown organically and would need a major refactoring.

You can either submit or sponsor an implementation.

**Comment by manticore-projects** [comment]
Fixed via [Tanish0019:fix-#1727](https://github.com/Tanish0019/JSqlParser/tree/fix-%231727)
